### PR TITLE
[proposal] hopefully easier to read changelog format

### PR DESCRIPTION
**Pull request by joshsleeper**
# problem
i was trying to read through the change log and it felt like more of a strain than it should be

# solution
took a quick swing at reformatting it in a way that makes a lot more sense to me and I think is much more readable. i tried to only make formatting changes and avoided reformatting the older, less consistently formatted changelog entries.

# aside
if it's hard to parse (hah) the change, try using the rich diff instead of the source diff. markdown reformatting is a pain to review in the source diff sometimes. 

also if you're not a fan, go ahead and snub my pr! 😄 it's your changelog and you're the one who has to keep updating it, so do whatever format you prefer. ojust figured I'd toss my two cents into a pr and see if you liked it. 👍 

# p.s.
i love this theme so much i can't even express, so thanks a ton for maintaining this.

### CODE REVIEW

**Comment by laCour** [comment]
Nice! I like this format much more. Thanks for the PR! 👍

**Comment by joshsleeper** [comment]
Of course, thanks again for the top-tier theme!
